ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```
@@ -713,6 +713,31 @@ tests:
     - chain: ipi-install-heterogeneous-day2-kerneltype
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-public-ipv4-pool-arm-f7
+  cron: 11 0 8,15,22,29 * *
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-public-ipv4-pool
```

Hey @yunjiang29 , I suggest to force the lease to `aws-3-quota-slice` which already have only  `us-east-1`, this is how I am setting in the dev job[1] to force always get the quota from region with support of BYO IP.

I am not 100% sure if this way is safe as new regions can be added to `aws-3-quota-slice`?

```suggestion
    leases:
    - env: LEASED_RESOURCE
      resource_type: aws-3-quota-slice
    workflow: cucushift-installer-rehearse-aws-ipi-public-ipv4-pool
```

[1] https://github.com/openshift/release/pull/51416/files#diff-b137b3cfb458afb6ff772856a362774409324d035d94f8436fcd3409a9406557R171-R173

---

@mtulio it's similar to QE's [aws-1-qe](https://github.com/openshift/release/blob/master/core-services/prow/02_config/generate-boskos.py#L15) profile, only us-east-1 slices will be added to this profile, for some particular tests, e.g. the initial version of LocalZone testing, Outpost testing, as they are available in us-east-1 only. 

> I am not 100% sure if this way is safe as new regions can be added to aws-3-quota-slice?

I think it's good for current tests, but in the future we could consider add such restriction to aws-3 or create separate profile for ip pool.

---

So actually this job will consume two slice quota, one is from `aws`, another is from `aws-3`, but the only the one from `aws-3` will be used?

---

@jianlinliu 
they are different lease resources:
IP pool consumes slice from `aws`, which is the only one that supports  IP pool [1][2]
Region/Cluster  consumes slice from `aws-3`

[1] https://github.com/openshift/ci-tools/blob/df898ffbd5c04ffe0720e2021f77d2465d6b89f3/pkg/steps/multi_stage/multi_stage.go#L379
[2] https://github.com/smg247/ci-tools/blob/9543639adec3a38d82b89fadf33c6e80c6351a76/pkg/api/types.go#L1926